components/shared/filters/modules/constants.ts

```diff
@@ -43,4 +43,5 @@ export const POPULAR_COLLECTIONS = {
     '2334273319', //Guardians and Creatures
     '1635680444', //KODACHAINS
   ],
+  ahk: ['u-2024', 'u-2023', '110', '30', '14', 'u-117', 'u-2022', 'u-31848'],
```

Comment:
I would add comments with description on collection name

Reply:
It's not needed because we have do sort for the collection by their names.

Reply:
I want to know which collection corresponds to these ids when I look at the code, please make it like other chains 🥺

Reply:
ok, will update it